src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
They also only use `true` and `false`. It's just written in a bit a confusing way. The first parameter of the `handleChange` method is just set using `bind` in the `onChange` callback directly.

@alexander-schranz I think if we do it like this the component almost doesn't make any sense at all, and it's harder to use than just handling a boolean...